File: app/src/main/java/com/cappielloantonio/play/model/RecentSearch.java
package com.cappielloantonio.play.model;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "recent_search")
public class RecentSearch {
    @NonNull
    @PrimaryKey
    @ColumnInfo(name = "search")
    private String search;

    public RecentSearch(@NonNull String search) {
        this.search = search;
    }

    @NonNull
    public String getSearch() {
        return search;
    }

    public void setSearch(@NonNull String search) {
        this.search = search;
    }
}

File: app/src/main/java/com/cappielloantonio/play/model/RecentSearch.kt
package com.cappielloantonio.play.model

import android.os.Parcelable
import androidx.annotation.Keep
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import kotlinx.android.parcel.Parcelize

@Keep
@Parcelize
@Entity(tableName = "recent_search")
data class RecentSearch(
    @PrimaryKey
    @ColumnInfo(name = "search")
    var search: String
) : Parcelable
